internal/servers/controller/handlers/authmethods/authmethod_service_test.go

```diff
@@ -570,6 +570,10 @@ func TestCreate(t *testing.T) {
 						lv, _ := structpb.NewList([]interface{}{"foo", "bar"})
 						return structpb.NewListValue(lv)
 					}(),
+					"claims_scopes": func() *structpb.Value {
```

Comment:
Consider also adding test coverage for the DefaultClaimsScope validation check.

Reply:
ty!